authentication/plaid_middleware.py

```diff
@@ -2,18 +2,6 @@
 Plaid setup
```

Comment:
Given that this no longer creates middleware, perhaps rename plaid_middleware.py to plaid_helper or something?